### PR TITLE
Add setter for ItemsToDrop in ItemsToDrop

### PR DESCRIPTION
**Pull request by antt0n**

### CODE REVIEW

File: Exiled.Events/EventArgs/DyingEventArgs.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class DyingEventArgs : EventArgs
         public DyingEventArgs(Player target, DamageHandlerBase damageHandler)
         {
             Handler = new CustomDamageHandler(target, damageHandler);
-            ItemsToDrop = new List<Item>(target.Items.ToList());
+            ItemsToDrop = target.ResetInventory(target.Items.ToList());
```
Comment (zhaguanyang) [inline Exiled.Events/EventArgs/DyingEventArgs.cs:34]: Have you even tried to compile it? ``ResetInventory`` is a void method, it doesn't return anything.

**Review by zhaguanyang — CHANGES_REQUESTED** [review]

**Comment by NaoUnderscore** [comment]
What does this PR mean?

**Comment by BuildBoy12** [comment]
I'm taking this as an actual joke considering it does not compile, let alone correspond with what the title says. Closing.

**Comment by zhaguanyang** [comment]
> What does this PR mean?

Cuz ItemsToDrop has a setter in the event args, while setting it does nothing.

**Pull request closed without merging**